core/services/gas/block_history_estimator_test.go

```diff
@@ -410,6 +411,7 @@ func TestBlockHistoryEstimator_Recalculate_NoEIP1559(t *testing.T) {
 		ethClient := cltest.NewEthClientMockWithDefaultChain(t)
 		config := newConfigWithEIP1559DynamicFeesDisabled(t)
 
+		config.On("ChainType").Return("")
```

Comment:
Generally if you have to do it in a lot of places like this, it might be worth adding it to the newConfigXXX functions

Reply:
Done